SchedulerJobs/SchedulerJobs.Services/BookingsApiService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.Extensions.Logging;
+using SchedulerJobs.Common.ApiHelper;
+using SchedulerJobs.Common.Configuration;
+using SchedulerJobs.Services.BookingApi.Contracts;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SchedulerJobs.Services
+{
+    public interface IBookingsApiService
+    {
+        Task<UserWithClosedConferencesResponse> GetUsersWithClosedConferences();
+        /// <summary>
+        /// Anonymises the hearing, case, person, organisation and participant data for hearings older than 3 months.
+        /// </summary>
+        Task AnonymiseHearings();
+    }
+    public class BookingsApiService : IBookingsApiService
+    {
+        private readonly HttpClient _httpClient;
+        private readonly ILogger _log;
+        private readonly ApiUriFactory _apiUriFactory;
+        public BookingsApiService(HttpClient httpClient, HearingServicesConfiguration hearingServicesConfiguration,
+            ILoggerFactory factory)
+        {
+            _httpClient = httpClient;
+            _log = factory.CreateLogger<BookingsApiService>();
```

Comment:
I think you have addeded the MS loggin nuget, so possibly just use ILogger<xxx> injected in